lib/event_store_client/connection.rb

```diff
@@ -79,5 +73,47 @@ def client
           host: host, port: port, per_page: per_page
         )
     end
+
+    def read_from_stream(stream, direction:, start:, count:, resolve_links:)
+      response =
+        client.read(
+          stream, start: start, direction: direction, count: count, resolve_links: resolve_links
+        )
+      return [] if response.body.nil? || response.body.empty?
+      JSON.parse(response.body)['entries'].map do |entry|
+        deserialize_event(entry)
+      end.reverse
```

Comment:
Stilll need reverse?

Reply:
Event Store returns events in order from the latest to the oldest. For me, it's more natural when I fetch events in order from the oldest to the latest. You apply events e.g. in a process manager starting from the oldest one. I want to get an event collection in the proper ordering from the event_store_client.